lib/config/configuration.js

```diff
@@ -718,6 +718,9 @@ Configuration.prototype.registerDefaultPresets = function() {
     // https://github.com/felixge/node-style-guide#nodejs-style-guide
     this.registerPreset('node-style-guide', require('../../presets/node-style-guide.json'));
 
+    // https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style#JavaScript_practices
```

Comment:
Not sure if linking directly to `#JavaScript_practices` is necessary, as most rules comes from the `#Naming_and_Formatting_code`-part